wikipedia_search_and_save/wikipedia_search_and_save.py

```diff
@@ -0,0 +1,63 @@
+import wikipedia
+from fpdf import FPDF
+
+
+class PDF(FPDF):
+    def chapter_title(self, label):
+        # Arial 12
+        self.set_font('Arial', '', 12)
+        # Background color
+        self.set_fill_color(200, 220, 255)
+        # Title
+        self.cell(0, 6, '%s' % (label), 0, 1, 'L', 1)
+        # Line break
+        self.ln(4)
+
+    def chapter_body(self, file_name):
+        # Read text file
+        with open(file_name, 'rb') as fh:
+            txt = fh.read().decode('latin-1')
+        # Times 12
+        self.set_font('Times', '', 12)
+        # Output justified text
+        self.multi_cell(0, 5, txt)
+        # Line break
+        self.ln()
+
+    def print_chapter(self, title, file_name):
+        self.add_page()
+        self.chapter_title(title)
+        self.chapter_body(file_name)
+
+
+def wikipedia_search(user_input):
+    query = wikipedia.page(user_input)
```

Comment:
And if it's possible to take input from the terminal using argparse library, please do that

Reply:
Yes, it's possible. I would modify the program to take the user input from the terminal using argparse.

Reply:
Hello @pawangeek 
Please check the latest update to this PR and confirm if that resolve this conversation. :)

Reply:
woohoo, you worked perfectly on this.